FiguresInheritance/Models/Circle.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FiguresInheritance.Models
+{
+    internal class Circle : Figure //Круг
+    {
+        protected float Radius { get; set; }
+        protected float Diameter { get; set; }
+        protected float Area { get; set; }
+
+        public Circle(float radius, float piValue = 3.14f)
```

Comment:
you should avoid magic numbers even as a default parameter. I suggest to use constant file(like one global file with all constants) or in some cases config files. Right there you can use Math.Pi instead

Reply:
It does not refer to the user somehow, and he doesnt need to think where this value comes from. This value gives from the programmer implementation as a default one. The user has he's own oppinion, and this way he can not touch the implementation, and just create a new object with a specific pi value. This way i want to tell that this value does not a "Magic number", and user dont really need to understand where this value comes from. It is not a required param, this way user can not touch it and it will work properly, or he can give he's own value, and constructor will process given value, not a magic number. I will fix it, but i hope you've understood my point of view.